tools/batchTestGenerator/testcases.json

```diff
@@ -118,6 +118,32 @@
 					"region": "us-west-2"
 				}
 			]
+		},
+		{
+			"type": "EKS_JAVA_INSTRUMENTATION_ADOT_OPERATOR",
```

Comment:
if you add the testcases to this file, they will be run by the collector. we need a new testcase file, that can live in the java repository. 

take a look into how the collector create the testcases: https://github.com/aws-observability/aws-otel-collector/blob/main/.github/workflows/CI.yml#L569

Reply:
@rapphil I noticed that all repos have their own copy of testcases.json (adot-operator, adot-collector) along with test-framework. but I couldn't understand when we do have argument to pick only certain services (like its done [here](https://github.com/aws-observability/aws-otel-collector/blob/bdeb7a6a4dba7c3a39813c1b1cee9db31c39fd31/.github/workflows/CI.yml#L51) for collector) then there should be no risk of collector running testcases of other services (like EKS_JAVA_INSTRUMENTATION_ADOT_OPERATOR), right? Wanted to check this with you?

Reply:
that is not a requirement, in fact, we should move the testcase.json file to the adot collector repository and keep the tool in the test framework repository. The inclusion mechanism in the ADOT collector workflow is used for the case we want to run tests in the dev branch.

Reply:
ok. in this case, should I update the `testcases.json` that we have in Collector repo or clone a new one in java-instrumentation repo?

Reply:
I concur, please add a `testcases.json` file in the `java-instrumentation` repo to generate a list of test cases. The collector does not currently do this but that's a deficiency in the collector.

Reply:
Removed changes in testcases.json and batchtestgenerator from here.